client/templates/initiative/mine/mine.js

```diff
@@ -1,3 +1,9 @@
 Template.initiativeMine.onRendered(function initiativeMineOnRendered() {
   this.$('ul.tabs').tabs();
+  $('body').addClass('my-initiative');
+});
+
+
+Template.initiativeHeader.onDestroyed(function(){
```

Comment:
Function needs a name and a space before `{`:

`Template.initiativeHeader.onDestroyed(function initiativeHeaderOnDestroyed() {`